profile.md

```diff
@@ -178,3 +179,98 @@ part of a Kubernetes API call:
   }
   ```
 
+## Service Metadata
+
+While the [specification](spec.md) does not mandate the property names used
+in the `metadata` objects, is it RECOMMENDED that the following names
+be used when possible in an attempt to provide some degree of interoperability
+and consistency.
+
+### Service Metadata Fields
+
+| Broker API Field | Type | Description |
+| --- | --- | --- |
+| name | CLI string | A short name for the service to be displayed in a catalog. |
+| description | string | A short 1-line description for the service, usually a single sentence or phrase. |
+| metadata.displayName | string | The name of the service to be displayed in graphical clients. |
+| metadata.imageUrl | string | The URL to an image. |
+| metadata.longDescription | string | Long description. |
+| metadata.providerDisplayName | string | The name of the upstream entity providing the actual service. |
+| metadata.documentationUrl | string | Link to documentation page for service. |
+| metadata.supportUrl | string | Link to support for the service. |
+
+### Plan Metadata Fields
+| Broker API Field | Type | Description |
+| --- | --- | --- |
+| name | CLI string | A short name for the service plan to be displayed in a catalog. |
+| description | string | A description of the service plan to be displayed in a catalog. |
+| metadata.bullets | array-of-strings | Features of this plan, to be displayed in a bulleted-list. |
+| metadata.costs | cost object | An array-of-objects that describes the costs of a service, in what currency, and the unit of measure. If there are multiple costs, all of them could be billed to the user (such as a monthly + usage costs at once). Each object MUST provide the following keys: `amount: { usd: float }, unit: string` This indicates the cost in USD of the service plan, and how frequently the cost is occurred, such as “MONTHLY” or “per 1000 messages”. |
```

Comment:
Should this just have type `object` since we do not define `cost object` anywhere (other than the information in the description on what keys _each object must provide_)?